src/h.js

```diff
@@ -1,37 +1,22 @@
 export default function (tag, data) {
   var node
-  var canConcat
-  var oldCanConcat
-
   var stack = []
   var children = []
 
   for (var i = arguments.length; i-- > 2;) {
-    stack.push(arguments[i])
+    stack[stack.length] = arguments[i]
```

Comment:
`array[array.length] = value` beats `array.push(value)`, but it's more bytes. This time I choose speed.